.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 Cargo.lock
+
+
```

Comment:
Don't add extra lines.

Reply:
I thought there was a failure due to a lack of carriage return in another Cargo.lock file.  I'll trim to just one

Reply:
Shouldn't the `Cargo.lock` be removed from `.gitignore` ??

Reply:
@haraldh Yes, probably.